packages/@coorpacademy-components/src/atom/checkbox/index.js

```diff
@@ -1,45 +1,31 @@
-import React from 'react';
+import React, {useState, useCallback} from 'react';
 import PropTypes from 'prop-types';
 import {noop} from 'lodash/fp';
 
-class Checkbox extends React.Component {
-  static propTypes = {
-    checked: PropTypes.bool,
-    onChange: PropTypes.func,
-    'data-name': PropTypes.string
-  };
+const Checkbox = props => {
```

Comment:
refactored this component because most of the propTypes where missing/ not explicit